src/styles/Fonts.ts

```diff
@@ -16,12 +15,10 @@ fontFace({
 
 const sharedFontStyles = {
   fontStyle: 'normal',
-  color: Colors.PrimaryText,
 };
 
 const sharedFontStylesDark = {
   fontStyle: 'normal',
-  color: Colors.PrimaryTextDark,
 };
```

Comment:
They all look the same now, shouldn't we have just one version of the fonts values, instead of Light vs Dark?

Reply:
That's right. I overlooked this. Thanks!